package/com.unity.formats.usd/Tests/Editor/ExportTests.cs

```diff
@@ -0,0 +1,102 @@
+using NUnit.Framework;
+using UnityEditor.SceneManagement;
+using UnityEngine;
+
+using USDScene = USD.NET.Scene;
+using UnityScene = UnityEngine.SceneManagement.Scene;
+using NUnit.Framework.Constraints;
+using USD.NET.Unity;
+
+namespace Unity.Formats.USD.Tests
+{
+    public class USDExportTests : BaseFixture
+    {
+        private string m_USDFilePath;
+        private USDScene m_USDScene;
+        private UnityScene m_UnityScene;
+
+        [SetUp]
+        public void SetUp()
+        {
+            m_UnityScene = EditorSceneManager.NewScene(NewSceneSetup.EmptyScene);
+            m_USDFilePath = CreateTmpUsdFile("USDExportTests.usda");
+            m_USDScene = USDScene.Create(m_USDFilePath);
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            m_UnityScene = EditorSceneManager.NewScene(NewSceneSetup.EmptyScene);
+            if (m_USDScene != null)
+            {
+                m_USDScene.Close();
+            }
```

Comment:
If we don't clean up USDExportTests.usda, could there be an error on a rerun of the file already existing when it is created in the setup?

Reply:
Good point, I will check this